SU-HSDO/suhumsci
# When an image is in a masonry, dont do lazy loading

# READY FOR REVIEW

# Summary
- Dont lazy load masonry images.

# Needed By (Date)
- ASAP

# Urgency
- HIGH

# Steps to Test
1. checkout this branch
1. `blt drupal:sync --site=archaeology --partial`
1. view `/research/publications` to ensure masonry appears to have applied correctly.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
